source/common/formatter/stream_info_formatter.cc

```diff
@@ -1743,6 +1744,21 @@ const StreamInfoFormatterProviderLookupTable& getKnownStreamInfoFormatterProvide
                     return absl::nullopt;
                   });
             }}},
+          {"DOWNSTREAM_NETWORK_NAMESPACE",
```

Comment:
The implementation seems to be what `%FILTER_STATE(envoy.network.network_namespace)%` would do? I suggested this as a replacement for the filter state, but if we already populate it, I think it makes more sense to just use it, and not introduce redundant formatter?

Reply:
I was thinking that if we want to share the data with upstream then it might be easier to have a formatter and we could simply populate a new filter state using the substitution formatter like, _"% DOWNSTREAM_NETWORK_NAMESPACE%"_ and share that key with the upstream.

I'll have to double check if it's possible for us to populate a new filter state with share with upstream on using the WK state. If that's possible then it's purely about ease of use and if we are gaining anything exposing this as a formatter.

Happy to close the PR out if we think that it's not required.

Reply:
ACK, should we close this out?